devicon.json

```
@@ -1,4 +1,29 @@
 [
+    {
+        "name": "aerospike",
+        "altnames": [
+            "aerospike"
+        ],
+        "tags": [
+            "data",
+            "database",
+            "nosql"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain"
+            ],
+            "font": [
+                "original",
+                "original-wordmark",
+                "plain"
+            ]
+        },
+        "color": "#C31618",
```

Update the color to `#c21417` as that is what's found in the original.svg and original-wordmark.svg.

```suggestion
        "color": "#c21417",
```